src/pages/about-us/about-us.js

```diff
@@ -1,128 +1,34 @@
 import React, { useEffect } from 'react';
-import Typography from '@material-ui/core/Typography';
+import { useDispatch, useSelector } from 'react-redux';
+import parse from 'html-react-parser';
 
 import { useStyles } from './about-us.style';
 import { ABOUT_US_IMAGES } from '../../configs';
+import { getBusinessPageByCode } from '../../redux/businessPages/businessPages.actions';
 
 const AboutUs = () => {
-  const styles = useStyles({ ...ABOUT_US_IMAGES });
+  const dispatch = useDispatch();
+  const { aboutUsPage, language } = useSelector(
+    ({ BusinessPages, Language }) => ({
+      aboutUsPage: BusinessPages.pages.aboutUs,
+      language: Language.language
+    })
+  );
 
   useEffect(() => {
     window.scrollTo(0, 0);
-  }, []);
+    dispatch(getBusinessPageByCode('about-us'));
+  }, [dispatch]);
+
+  const aboutUsText = Object.keys(aboutUsPage).length
```

Comment:
Do we really need this check? I think business pages should always be translated, it should be ensured on the backend and admin side. Here we should not care about it